README.md

```diff
@@ -1,95 +1,99 @@
-2. Dual-redundancy IMU components (onboard HMC5883L compass)
-3. Support PPM/SBUS/SPEKTRUM DSM/Satellite RC input
-4. Support FrSky/Wifi telemetry
-5. One extra I2C port (so can support external compass and px4flow simultaneously)
-6. Secondary USB port  for companion computer (converted from UART2 using onboard CP2104)
-7. nrf/SPI port for SPI devices
-8. exposed SWD debug port
-9. Built-in vibration dampening for isolated IMU, support hard mount on air frames.
-10. LTC2875 as CAN controller (+/- 60V overvoltage fault protection)
+#### FMU Versions
 
-MindRacer V1.2 board (Hardware version: v1.2)
+The Pixhawk project has evolved the FMU design through a number of verisons.
 
-MindRacer specification:
-http://mindpx.net/assets/accessories/mindracer_spec_v1.2.pdf
+These are named using the designation: FMUvX (e.g.: FMUv1, FMUv2, FMUv3, FMUv4, etc.).
+Higher FMU numbers indicate that the board is more recent, but may not indicate increased capability (versions can be almost identical - differing only in connector wiring).
 
-MindRacer user manual:
-http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf
+The designs listed below (with a high level overview of the main differences).
 
-Key features of v1.2:
+Version | Description | Design | Hardware | Description
+--- | ---
+[FMUv1](FMUv1/README.md) & [IOv1](IOv1) | (Discontinued) Original Flight Management Unit and Separate I/O board.
+[FMUv2](FMUv2/README.md) | Single board with STM32427VI processor.
+[FMUv3](FMUv3_REV_D/README.md)| Identical to FMUv2, but usable flash doubled to 2MB.
+[FMUv4](FMUv4/README.md) | Increased RAM. Faster CPU. More serial ports. No IO processor
+FMUv4-PRO | Slightly increased RAM. More serial ports. IO processor.
+[FMUv5](FMUv5/README.md) | New processor (F7). Much faster. More RAM. More CAN busses. Much more configurable.<br>> **Note** Minimum specification provided (pinout info, but no schematics).
 
-1. 35 x 35mm (30.5 mm mounting holes) ultra mini size, weighs about 6g
-2. STM32F427 as processor
-3. 6 main PWM outputs
-4. onboard MPU6500 & HMC5883L
-5. dual 2x7 stackable header pins for I/O port replication/expansion
-6. optional IMU isolation mounting support
-7. Support PPM/SBUS/SPEKTRUM DSM/Satellite RC input
-8. Support FrSky/Wifi telemetry
-9.  onboard RTC battery and TF card slot
-10. DroneCode standard connectors
 
-Note: MindPX v2.8 & MindRacer v1.2 share same firmware.
 
-[Sapog ESC](https://github.com/PX4/sapog) Hardware
-==================================================
+### Sapog ESC
 
-The directory `sapog_reference_hardware` contains the reference hardware design for the
-Sapog ESC firmware. Refer to its README.md for more info.
+- [Sapog ESC](https://github.com/PX4/sapog) Hardware - The directory [sapog_reference_hardware](sapog_reference_hardware/README.md) contains the reference hardware design for the Sapog ESC firmware.
 
-Creative Commons License Disclaimer
-===================================
+### Other
```

Comment:
@bkueng Do you think you can tell me what these folders are for. I assume flow is PX4FLOW. Where are the other ones used?

Reply:
I don't know any of the other ones either.